docs/src/documentation/03-components/02-structure/card/03-accessibility.mdx

```diff
@@ -0,0 +1,139 @@
+| --------------- | :------- | :------------------------------------------------------------------------------------------------------------------------------- |
+| labelClose      | `string` | Label for the close button that's announced to screen readers, ensuring users understand the button's purpose.                   |
+| dataA11ySection | `string` | Optional identifier for the Card that can be used to improve screen reader navigation and identification of content sections.    |
+| titleAs         | `enum`   | Controls the semantic heading level (h1-h6) of the Card title, helping maintain proper document structure and heading hierarchy. |
+
+### CardSection accessibility props
+
+| Name    | Type   | Description                                                                                                    |
+| ------- | :----- | :------------------------------------------------------------------------------------------------------------- |
+| titleAs | `enum` | Controls the semantic heading level of the CardSection title, allowing proper document hierarchy within Cards. |
+
+### Keyboard navigation
+
+#### Card component
+
+Cards themselves are not focusable elements, but they may contain interactive elements like buttons, links, or form controls that are part of the card's content. These interactive elements follow standard keyboard navigation:
+
+- Interactive elements within cards can be accessed using the `Tab` key
+- When a Card contains the close button, it can be activated using `Enter` or `Space`
+
+#### CardSection component
+
+CardSection components have specific keyboard behaviors when they're expandable:
+
+- Expandable sections can be toggled with `Enter` or `Space` when focused
+- Focus is managed automatically when expanding/collapsing sections
+- When multiple expandable sections exist in a Card, users can navigate between them using `Tab`
+
+### Screen reader experience
+
+#### Card component
+
+Screen readers will announce:
+
+- The Card title as specified by the `titleAs` prop (when provided)
+- The close button with the custom label specified in `labelClose` prop
+- The section identifier provided in `dataA11ySection` (when provided)
+
+#### CardSection component
+
+Screen readers will announce:
+
+- The CardSection title as specified by the `titleAs` prop
+- The expanded/collapsed state for expandable sections
+- The content within the section when expanded
+
+### Important restrictions
+
+- **CardSection with actions and expandability**: When using the `actions` prop with an expandable CardSection (`expandable={true}` or with `onClick` handler), prop `actions` should be undefined or passed a non-interactive element. This prevents focus management issues and keyboard navigation confusion. Interactive elements in this context can create conflicts with the expansion functionality, leading to poor accessibility.
```

Comment:
> When using the `actions` prop with an expandable CardSection (`expandable={true}` or with `onClick` handler), prop `actions` should be undefined or passed a non-interactive element.

I find this sentence quite confusing. How can I be using it but set it to undefined?